docs/UserGuide.md

```diff
@@ -89,16 +89,20 @@ CS2103T Laboratory 10:00-12:00
 
 ### Adding a default course : `teach`
 
-Adds a default course to the address book.
+The default course is usually the course that you are currently teaching.
```

Comment:
The language here is not consistent with the other commands

Reply:
resolved in [b3a68ec](https://github.com/AY2324S1-CS2103T-T10-1/tp/pull/123/commits/b3a68ec98c865c793111441f1611b3629c26c262)